client/volume_list.go

```diff
@@ -8,18 +8,28 @@ import (
 	"github.com/moby/moby/api/types/volume"
 )
 
+// VolumeListOptions holds parameters to list volumes.
+type VolumeListOptions struct {
+	Filters Filters
+}
+
+// VolumeListResponse holds the result from the [Client.VolumeList] method.
+type VolumeListResponse struct {
+	volume.ListResponse
+}
```

Comment:
I went a few times around between aliasing, embedding, and wrapping. I guess the issue specified wrapping but lmk.

Reply:
Yeah I think embedding makes sense in this case.